main.tf

```diff
@@ -128,10 +128,11 @@ resource "aws_instance" "default" {
 
 module "ssh_key_pair" {
   source                = "cloudposse/key-pair/aws"
-  version               = "0.18.0"
+  version               = "0.18.1"
   ssh_public_key_path   = local.ssh_key_pair_path
   private_key_extension = ".pem"
   generate_ssh_key      = var.generate_ssh_key_pair
```

Comment:
Couldn't this be currently achieved by passing in ssh_key_pair with generate_ssh_key_pair to false?